packages/react/src/publication/usePublications.ts

```diff
@@ -15,6 +15,7 @@ export type UsePublicationsArgs = PaginatedArgs<
   WithObserverIdOverride<{
     metadataFilter?: PublicationMetadataFilters;
     profileId: ProfileId;
+    publicationTypes?: PublicationTypes | PublicationTypes[];
```

Comment:
Wondering if we could force consumers to always provider an array?

Reply:
Could you also review all field policies if they include `publicationTypes` as a cache key to have separate cache storages in case the query is called with different `publicationTypes` 🙏 ?

Reply:
regarding field policies, is this what you mean? https://github.com/lens-protocol/lens-sdk/blob/04647bbeb13fc18bcd030f4abfac6b8e15d82439/packages/api-bindings/src/apollo/cache/createPublicationsFieldPolicy.ts 
seems `publicationTypes` is already there

Reply:
yes, that was exactly what I was not sure about and wanted to double check. Thanks.